core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1341,6 +1374,24 @@ public function getAlias()
 		return $arrAlias;
 	}
 
+	/**
+	 * Saves the article alias
+	 *
+	 * @param mixed         $varValue
+	 * @param DataContainer $dc
+	 *
+	 * @return mixed
+	 */
+	public function saveAlias($varValue, DataContainer $dc)
```

Comment:
This method seems duplicate?

Reply:
It's not, it's `$dc->activeRecord->pid` vs. `$dc->activeRecord->id`